x/profiles/keeper/handler.go

```diff
@@ -140,3 +149,88 @@ func handleMsgDeleteProfile(ctx sdk.Context, keeper Keeper, msg types.MsgDeleteP
 
 	return &result, nil
 }
+
+// handleMsgRequestDTagTransfer handles the request of a dTag transfer
+func handleMsgRequestDTagTransfer(ctx sdk.Context, keeper Keeper, msg types.MsgRequestDTagTransfer) (*sdk.Result, error) {
+	transferRequest := types.NewDTagTransferRequest(msg.CurrentOwner, msg.ReceivingUser)
```

Comment:
What if we also save the DTag that is the object of the trade? This way we can later check to make sure it hasn't been edited.